packages/rancher-istio/charts/configs/istio-base.yaml

```diff
@@ -52,7 +52,7 @@ spec:
       enabled: {{ .Values.policy.enabled }}
     telemetry:
       enabled: {{ .Values.telemetry.enabled }}
-  hub: docker.io/istio
+  hub: {{ .Values.global.hub | default "docker.io/istio" }}
```

Comment:
Values.global.hub should be the system_default_registry helper ? and default to docker.io/rancher

Reply:
You are correct, I will update to use the registry helper. I guess I am not convinced on what the default should be - does it matter if they want to use the chart outside of rancher and it still point to our repo? I guess they could always change it with an overlay file of their own 🤔

Reply:
Because of our setup it will always use our orgs images, they likely would never specify a registry so that would just be empty string

Reply:
instead of using the `default` templating option, I updated the helper method to set the default value instead of being empty that way all the logic is in one spot

Reply:
Hmm, I suppose thats ok. Could see this having weird consequences but we can try it out.

Reply:
Actually that won't work because this hub value is supposed to have both registry and org in it. Our helper should only do the registry, otherwise you'd get: `docker.io/rancher/rancher/kiali-kiali`

Reply:
The way hub is used throughout the manifests is `{{ .Values.global.hub }}{{ .Values.pilot.image }}:{{ .Values.tag }}` which gives us the flexibility to have 1. hub be `<registry>` and repository be `<org/image>` or 2. hub be <registry/org> and repository be `<image>` so it might come down to what is easier for us to traverse in the values.yaml for the image.txt file. 

Right now we are doing 1. so it feels weird to have hub be `<registry>` but then default to `<registry/org>` because that will also result in `docker.io/rancher/rancher/kiali-kiali` if registry is not set. If we stick with 1. I will default to `docker.io` and I will set that in the registry helper, since it is just setting the registry